data/json/overmap/overmap_terrain/overmap_terrain.json

```diff
@@ -39,7 +39,7 @@
     "extras": "build",
     "mondensity": 2,
     "see_cost": "high",
-    "flags": [ "RISK_HIGH", "GENERIC_LOOT" ]
+    "flags": [ "RISK_HIGH", "GENERIC_LOOT", "PP_GENERATE_RIOT_DAMAGE" ]
```

Comment:
This aims to only work in cities (`generic_city_building_no_sidewalk`), but it ends up working in the countryside as well because `generic_rural_building` is `"copy-from": "generic_city_building_no_sidewalk",`
And there are also a bunch of random overmap terrains that are not really "city" but copy-from this one anyway (`fire lookout tower`, `golf course`, `lumbermill_dforest` (`"name": "forest",) ), and riots should not have affected those.

(i'm writing this as a comment here where nobody will see it, because I'm too tired to make a proper bug report, yes)